tests/utils/get-text-up-until-caret-in-element.test.ts

```
@@ -0,0 +1,64 @@
+import { getTextUptilCaretInElement } from '@/utils/get-text-up-until-caret-in-element';
```

ESLint

[error] Expected 2 empty lines after import statement not followed by another import.

(import/newline-after-import)